### ISSUE TITLE
Adding the fields to the sales order api endpoint

### ISSUE THREAD

**Issue by paulh1992** [sbodak/magento2-checkout-custom-form#29]
Hi, would it be possible to add the custom fields to the sales order api endpoint on magento 2 please?

when doing a get request to this /rest/all/V1/orders/000000028

we don't see the fields at all

**Comment by sbodak** [comment]
@paulh1992 that's true - there is no extension attributes added for them. In the future, I will add it but right not, unfortunately, I don't have enough time.

**Comment by paulh1992** [comment]
Managed to get someone to enhance it for us anyway :)
